bentoml/_internal/utils/analytics/usage_stats.py

```diff
@@ -204,27 +250,33 @@ def track_serve(
         yield
         return
 
-    _track_serve_init(svc, production)
+    _track_serve_init(svc=svc, production=production, grpc=grpc)
 
     if _usage_event_debugging():
         tracking_interval = 5
     else:
-        tracking_interval = SERVE_USAGE_TRACKING_INTERVAL_SECONDS
+        tracking_interval = SERVE_USAGE_TRACKING_INTERVAL_SECONDS  # pragma: no cover
 
     stop_event = threading.Event()
 
     @silent
     def loop() -> t.NoReturn:  # type: ignore
         last_tracked_timestamp: datetime = serve_info.serve_started_timestamp
         while not stop_event.wait(tracking_interval):  # pragma: no cover
+            metrics_type = "not_available"
+            metrics, use_legacy_metrics = get_metrics_report(metrics_client, grpc=grpc)
+            if use_legacy_metrics is not None:
+                metrics_type = "legacy" if use_legacy_metrics else "current"
```

Comment:
I'm not sure if we want to call the metrics type `legacy` and `current`. Maybe we can call it v1 and v2? @ssheng